ttmar_starter.html

```diff
@@ -60,6 +60,45 @@
 
 
   <script>
+    var paragraph1 = document.getElementById("first_paragraph");
+    paragraph1.addEventListener("click", function() {
+      document.body.style.background = "#ccc";
+    })
+
+    var paragraph2 = document.getElementById("second_paragraph");
+    paragraph2.addEventListener("click", function() {
+      document.querySelector(".second_paragraph").style.display = "block";
+    })
+
+    var paragraph3 = document.getElementById("first_chorus");
+    paragraph3.addEventListener("click", function(event) {
+      event.preventDefault(event);
+      document.querySelector(".first_chorus").style.display = "block";
+    })
+
+    var paragraph4 = document.getElementById("third_paragraph");
+    paragraph4.addEventListener("click", function(event) {
+      event.preventDefault(event);
+      var thirdArray = document.querySelectorAll(".third_paragraph");
```

Comment:
I'd suggest thinking a bit about variable names. I realize this is somewhat encouraged by the markup (class names), but it's a bit confusing to see `paragraph4` and `thirdArray`. If at all possible, I'd try to find a way to make them more consistent.

Also, I try to avoid using `array` in a var name when possible. I might suggest `var thirdParagraphElements` here to convey that it contains elements (the plural implies an array).